controllers/operator/nmstate_controller_test.go

```diff
@@ -148,8 +148,8 @@ var _ = Describe("NMState controller reconcile", func() {
 		)
 		BeforeEach(func() {
 			request.Name = existingNMStateName
-			crdsPath := filepath.Join(manifestsDir, "kubernetes-nmstate/crds/")
-			dir, err := ioutil.ReadDir(crdsPath)
+			crdsPath := filepath.Join(manifestsDir, "kubernetes-nmstate", "crds")
```

Comment:
Yes, the linter advises to avoid "/". Which is exactly what I did. I probably did not get what you wanted to say, is there something you would change?

Reply:
Huh, I guess I read this as `"kubernetes-nmstate/crds"` o_O, disregard my comment